### ISSUE TITLE
Encoding error on notebook upload

### ISSUE THREAD

**Issue by rafaelhbarros** [Anaconda-Server/anaconda-client#277]
When uploading a notebook with diacritic, this happens:

![2016-02-01](https://cloud.githubusercontent.com/assets/3138550/12726424/1fa83e4e-c8dd-11e5-9472-5d326327ec31.png)

**Comment by srossross** [comment]
@rafaelhbarros can you attach an offending notebook?

**Comment by rafaelhbarros** [comment]
```
$ anaconda notebook upload não.ipynb
Using Anaconda Cloud api site https://api.anaconda.org
[ERROR] 
Traceback (most recent call last):
  File "/Users/rdebarros/miniconda2/envs/anaconda.org/bin/anaconda", line 6, in <module>
    sys.exit(main())
  File "/Users/rdebarros/miniconda2/envs/anaconda.org/lib/python2.7/site-packages/binstar_client/scripts/cli.py", line 87, in main
    description=__doc__, version=version)
  File "/Users/rdebarros/miniconda2/envs/anaconda.org/lib/python2.7/site-packages/binstar_client/scripts/cli.py", line 66, in binstar_main
    return args.main(args)
  File "/Users/rdebarros/miniconda2/envs/anaconda.org/lib/python2.7/site-packages/binstar_client/commands/notebook.py", line 111, in upload
    upload_info = uploader.upload(force=args.force)
  File "/Users/rdebarros/miniconda2/envs/anaconda.org/lib/python2.7/site-packages/binstar_client/utils/notebook/uploader.py", line 39, in upload
    self.package and self.release
  File "/Users/rdebarros/miniconda2/envs/anaconda.org/lib/python2.7/site-packages/binstar_client/utils/notebook/uploader.py", line 94, in package
    self._package = self.aserver_api.package(self.username, self.project)
  File "/Users/rdebarros/miniconda2/envs/anaconda.org/lib/python2.7/site-packages/binstar_client/utils/notebook/uploader.py", line 68, in project
    return re.sub('\-ipynb$', '', parameterize(os.path.basename(self.filepath)))
  File "/Users/rdebarros/miniconda2/envs/anaconda.org/lib/python2.7/site-packages/binstar_client/utils/notebook/inflection.py", line 18, in parameterize
    string = transliterate(string)
  File "/Users/rdebarros/miniconda2/envs/anaconda.org/lib/python2.7/site-packages/binstar_client/utils/notebook/inflection.py", line 43, in transliterate
    normalized = unicodedata.normalize('NFKD', unicode(string))
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 1: ordinal not in range(128)
```
Notebook attached in zip cause github doens't support ipynb:

[não.ipynb.zip](https://github.com/Anaconda-Server/anaconda-client/files/169762/nao.ipynb.zip)

**Comment by srossross** [comment]
thanks

**Comment by srossross** [comment]
@bollwyvl this will also be fixed with #1583 right?

**Comment by srossross** [comment]
This will be fixed with #306
